src/api/java/mekanism/api/MekanismIMC.java

```diff
@@ -192,7 +334,7 @@ public record ModuleContainerTarget(Item container, String imcMethod) {
          * @param container Item that will be the module container.
          * @param imcMethod Method used to add modules as supported to the container.
          */
-        public ModuleContainerTarget(ItemLike container, String imcMethod) {
+        public ModuleContainerTarget(ItemLike container, String imcMethod) {//TODO - 1.22: Switch this to taking a Holder<Item>
```

Comment:
can this not be done now?

Reply:
It is technically a tiny breaking change as I want to switch ModuleContainerTarget's record to use `Holder<Item>`, but as we are in alpha, and also [literally no one except us is trying to call the container method](https://waifu.neoforged.net/d/aec29a5ymel1cc/waifu-classes?orgId=1&from=now-1y&to=now&timezone=browser&var-version=1.21.1-neoforge&var-class=mekanism%2Fapi%2FMekanismIMC$ModuleContainerTarget&var-method=.%2A&var-field=.%2A) I am just going to go ahead and do it now.